target/linux/mediatek/patches-5.10/900-dts-mt7622-bpi-r64-aliases-for-dtoverlay.patch

```diff
@@ -78,3 +78,38 @@
 +		};
 +	};
 +};
+--- /dev/null
++++ b/arch/arm64/boot/dts/mediatek/mt7622-bananapi-bpi-r64-qorvo-dw3000.dts
+@@ -0,0 +1,22 @@
++/* SPDX-License-Identifier: (GPL-2.0-only OR MIT) */
++
++#include <dt-bindings/gpio/gpio.h>
```

Comment:
You aren't using any symbols from `gpio.h`, are you?

Reply:
nope, forgot to delete that.